vision/google/cloud/vision/_gax.py

```diff
@@ -30,24 +30,25 @@ def __init__(self, client=None):
         self._client = client
         self._annotator_client = image_annotator_client.ImageAnnotatorClient()
 
-    def annotate(self, image, features):
+    def annotate(self, images):
```

Comment:
@daspecster 

1. Your docstring needs to make this clear.
1. You should not make breaking interface changes on unrelated PRs